<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedPlus Clinic - Advanced Healthcare System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #0f2027, stop:0.5 #203a43, stop:1 #2c5364);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameCircle1">
    <property name="geometry">
     <rect>
      <x>-100</x>
      <y>-100</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    background: qradialgradient(cx:0.5, cy:0.5, radius:1,
        fx:0.5, fy:0.5, stop:0 rgba(66, 165, 245, 0.3), stop:1 transparent);
    border-radius: 200px;
    border: none;
}</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameCircle2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>600</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    background: qradialgradient(cx:0.5, cy:0.5, radius:1,
        fx:0.5, fy:0.5, stop:0 rgba(102, 187, 106, 0.3), stop:1 transparent);
    border-radius: 250px;
    border: none;
}</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameBranding">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>851</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 rgba(25, 118, 210, 0.95),
        stop:1 rgba(13, 71, 161, 0.95));
    border: none;
}</string>
    </property>
    <widget class="QLabel" name="lblBrandIcon">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background: qradialgradient(cx:0.5, cy:0.5, radius:0.8,
        fx:0.5, fy:0.5,
        stop:0 rgba(255, 255, 255, 0.2),
        stop:0.7 rgba(255, 255, 255, 0.1),
        stop:1 transparent);
    border: 5px solid rgba(255, 255, 255, 0.3);
    border-radius: 150px;
    font-size: 150px;
    color: white;
    qproperty-alignment: AlignCenter;
}</string>
     </property>
     <property name="text">
      <string>üè•</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblBrandName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>390</y>
       <width>600</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 56px;
    font-weight: bold;
    color: white;
    background: transparent;
    qproperty-alignment: AlignCenter;
    letter-spacing: 2px;
}</string>
     </property>
     <property name="text">
      <string>MedPlus</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTagline">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>500</y>
       <width>600</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 20px;
    color: rgba(255, 255, 255, 0.9);
    font-style: italic;
    background: transparent;
    qproperty-alignment: AlignCenter;
    letter-spacing: 1px;
}</string>
     </property>
     <property name="text">
      <string>Advanced Healthcare Management System</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblFeature1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>550</y>
      <width>400</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 16px;
    color: rgba(255, 255, 255, 0.85);
    background: transparent;
    padding-left: 10px;
}</string>
    </property>
    <property name="text">
     <string>‚úì Easy Online Appointment Booking</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFeature2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>590</y>
      <width>400</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 16px;
    color: rgba(255, 255, 255, 0.85);
    background: transparent;
    padding-left: 10px;
}</string>
    </property>
    <property name="text">
     <string>‚úì 24/7 Access to Medical Records</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFeature3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>630</y>
      <width>400</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 16px;
    color: rgba(255, 255, 255, 0.85);
    background: transparent;
    padding-left: 10px;
}</string>
    </property>
    <property name="text">
     <string>‚úì Secure &amp; Confidential</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFeature4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>670</y>
      <width>400</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 16px;
    color: rgba(255, 255, 255, 0.85);
    background: transparent;
    padding-left: 10px;
}</string>
    </property>
    <property name="text">
     <string>‚úì Expert Doctors Available</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameLoginPanel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>851</width>
      <height>851</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    background-color: #ffffff;
    border: none;
}</string>
    </property>
    <widget class="QLabel" name="lblFooter">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>780</y>
       <width>500</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 12px;
    color: #9e9e9e;
    background: transparent;
    qproperty-alignment: AlignCenter;
}</string>
     </property>
     <property name="text">
      <string>¬© 2025 MedPlus Clinic. All rights reserved.</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblWelcome">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>120</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 42px;
    font-weight: bold;
    color: #1565c0;
    background: transparent;
    qproperty-alignment: AlignLeft;
}</string>
    </property>
    <property name="text">
     <string>Welcome Back!</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSubtitle">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>185</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 16px;
    color: #757575;
    background: transparent;
    qproperty-alignment: AlignLeft;
}</string>
    </property>
    <property name="text">
     <string>Sign in to continue to your account</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUsernameIcon">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>270</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 15px;
    font-weight: 600;
    color: #37474f;
    background: transparent;
}</string>
    </property>
    <property name="text">
     <string>üë§ Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>310</y>
      <width>500</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    font-size: 16px;
    color: #212121;
    padding: 15px 20px;
    border: 2px solid #e0e0e0;
    border-radius: 12px;
    background-color: #fafafa;
}
QLineEdit:focus {
    border: 2px solid #1976d2;
    background-color: #ffffff;
}</string>
    </property>
    <property name="placeholderText">
     <string>Enter your username</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPasswordIcon">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>400</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 15px;
    font-weight: 600;
    color: #37474f;
    background: transparent;
}</string>
    </property>
    <property name="text">
     <string>üîí Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>440</y>
      <width>500</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    font-size: 16px;
    color: #212121;
    padding: 15px 20px;
    border: 2px solid #e0e0e0;
    border-radius: 12px;
    background-color: #fafafa;
}
QLineEdit:focus {
    border: 2px solid #1976d2;
    background-color: #ffffff;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Enter your password</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblForgotPassword">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>520</y>
      <width>220</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 13px;
    color: #1976d2;
    background: transparent;
    qproperty-alignment: AlignRight;
}</string>
    </property>
    <property name="text">
     <string>Forgot Password?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>580</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 18px;
    font-weight: bold;
    color: white;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #1976d2, stop:1 #42a5f5);
    border: none;
    border-radius: 12px;
    padding: 15px;
}
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #1565c0, stop:1 #1976d2);
}
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #0d47a1, stop:1 #1565c0);
}</string>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
   <widget class="QFrame" name="lineDivider">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>680</y>
      <width>500</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    background-color: #e0e0e0;
    border: none;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRegisterText">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>710</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 14px;
    color: #757575;
    background: transparent;
}</string>
    </property>
    <property name="text">
     <string>Don't have an account?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regLogin">
    <property name="geometry">
     <rect>
      <x>985</x>
      <y>705</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 14px;
    font-weight: 600;
    color: #1976d2;
    background: transparent;
    border: 2px solid #1976d2;
    border-radius: 8px;
    padding: 8px 16px;
}
QPushButton:hover {
    background-color: #e3f2fd;
    border-color: #0d47a1;
}
QPushButton:pressed {
    background-color: #bbdefb;
}</string>
    </property>
    <property name="text">
     <string>Register Now</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>705</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 14px;
    font-weight: 600;
    color: #757575;
    background: transparent;
    border: 2px solid #bdbdbd;
    border-radius: 8px;
    padding: 8px 16px;
}
QPushButton:hover {
    background-color: #f5f5f5;
    border-color: #757575;
}
QPushButton:pressed {
    background-color: #e0e0e0;
}</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
